api/app/views/notification_mailer/comment_notification.txt.erb

```diff
@@ -0,0 +1,17 @@
+<%
+  if @comment.subject.is_a? Resource
```

Comment:
More logic than I'd like to see in an email template. Why are we building URL string here, and in other places building them in models? We need to be consistent.